lib/facility_factory.rb

```diff
@@ -0,0 +1,36 @@
+class FacilityFactory
+
+  def create_facilities(office_locations)
+    office_locations.map do |office_location|
+      office_location[:name] = office_location[:dmv_office]
+      office_location[:address] = office_location[:address_li]
+      Facility.new(office_location)
+    end
+  end
+
+  # def concatenate_address
```

Comment:
Something tricky about this factory -- the facility data that were reading in has a list of services the facility provides! So, instead of starting `services` as an empty array, you could refactor your Facility initialize to pull in the existing services from the data